src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate(callback, value) {
+      if (typeof value === 'number') {
+        this.result = callback(value, this.result);
+
+        return this;
+      }
+    },
+    add(value, result) {
+      return result + value;
+    },
+    subtract(value, result) {
+      return result - value;
+    },
+    divide(value, result) {
+      if (value !== 0) {
+        return result / value;
+      }
+
+      return 'error';
```

Comment:
Need to use `throw new Error` and you need to use more informative text for the error, the user should understand in detail what the error is